build.gradle

```diff
@@ -72,6 +72,7 @@ propertyDefaultIfUnset("usesShadowedDependencies", false)
 propertyDefaultIfUnset("minimizeShadowedDependencies", true)
 propertyDefaultIfUnset("relocateShadowedDependencies", true)
 propertyDefaultIfUnset("separateRunDirectories", false)
+propertyDefaultIfUnset("versionDisplayFormat", '$VERSION')
```

Comment:
Should set this to the gradle.properties default (or similar, if the special char doesn't play nice) since this is a new property won't be pulled by a script update, and this isn't a great default